apps/myaccount/src/components/account-recovery/options/index.ts

```diff
@@ -18,3 +18,4 @@
 
 export * from "./email-recovery";
 export * from "./security-questions-recovery";
+export * from "./sms-recovery/sms-recovery";
```

Comment:
we are moving away from index files right ?
Then shall we let this new "sms-recovery" file to be directly imported instead of using index file ?

cc: @JayaShakthi97

Reply:
Since this index file is already here, wouldn't it make sense to use it instead of using direct imports for this file only?